superset-frontend/src/components/Menu/LanguagePicker.tsx

```diff
@@ -17,15 +17,17 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { NavDropdown, MenuItem } from 'react-bootstrap';
 
-const propTypes = {
-  locale: PropTypes.string.isRequired,
-  languages: PropTypes.object.isRequired,
-};
+export interface LanguagePickerInterface {
+  locale: string;
+  languages: object;
```

Comment:
Instead of object, we prefer `Record<string, any>`. Although in this case, perhaps we can type this even better, either by looking at what the backend returns or by looking at the code below. It seems like we might be able to type this as something like:
```typescript
languages: { 
  [key: string]: {
    flag: string;
    url: string;
    name: string;
  };
};
```

Reply:
Yes. added the definitions. thanks.